internal/pkg/addon/storage.go

```diff
@@ -15,11 +15,19 @@ import (
 )
 
 const (
-	dynamoDbTemplatePath = "addons/ddb/cf.yml"
-	s3TemplatePath       = "addons/s3/cf.yml"
-	rdsTemplatePath      = "addons/aurora/cf.yml"
-	rdsRDWSTemplatePath  = "addons/aurora/rdws/cf.yml"
-	rdsRDWSParamsPath    = "addons/aurora/rdws/addons.parameters.yml"
+	dynamoDbTemplatePath  = "addons/ddb/cf.yml"
+	s3TemplatePath        = "addons/s3/cf.yml"
+	rdsTemplatePath       = "addons/aurora/cf.yml"
+	rdsV2TemplatePath     = "addons/aurora/serverlessv2.yml"
+	rdsRDWSTemplatePath   = "addons/aurora/rdws/cf.yml"
+	rdsRDWSV2TemplatePath = "addons/aurora/rdws/serverlessv2.yml"
+	rdsRDWSParamsPath     = "addons/aurora/rdws/addons.parameters.yml"
+)
+
+const (
+	// Aurora Serverless versions.
+	ServerlessVersionV1 = "V1"
```

Comment:
Maybe we should lowercase the 'v' (here, flag description, etc.), based on https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-serverless-v2.html and Go/Semantic Versioning conventions. The flag should accept both upper and lower case input, though.

Reply:
I lowercased the `serverless-version` flag to `v1` or `v2`.

> The flag should accept both upper and lower case input, though.

I think the flag should accept only lower case for now. The database engine flag currently accepts only "MySQL" or "PostgreSQL", not "mysql" nor "postgresql" and `serverless-version` flag should do the same.